packages/grid/src/key-handlers/SelectionKeyHandler.ts

```diff
@@ -279,7 +286,15 @@ class SelectionKeyHandler extends KeyHandler {
       isShiftKey,
       false
     );
-    grid.setViewState({ top: viewportPosition });
+
+    const stickyOptions: StickyOptions = {
+      shouldStickBottom:
+        event.key === 'ArrowDown' ||
+        event.key === 'End' ||
+        event.key === 'PageDown',
+      shouldStickRight: event.key === 'ArrowRight',
```

Comment:
We shouldn't be doing these checks again - we're already in the `handlePageDown` method, which is triggered only on a keyboard event when navigating page down. Just do `stickOptions = { shouldStickBottom: true, shouldStickRight: false }`